front/app/containers/App/index.tsx

```diff
@@ -279,10 +279,17 @@ const App = ({ children }: Props) => {
   const isIdeaEditPage = isPage('idea_edit', location.pathname);
   const isEventPage = isPage('event_page', location.pathname);
   const isNativeSurveyPage = isPage('native_survey', location.pathname);
+  const isNewIdeaPage = isPage('idea_form', location.pathname);
+  const isEditIdeaPage = isPage('idea_edit', location.pathname);
```

Comment:
You have both `isIdeaEditPage` and `isEditIdeaPage` which seem to be the same thing